async/as/Thread.h

```diff
@@ -85,11 +87,12 @@ class Thread : public ThreadBase<Executor, InitParameter, Parameter>
 
 	void setExecutor(Executor &executor)
 	{
+#ifndef __APPLE__
 		ThreadBase<Executor, InitParameter, Parameter>::setExecutor(executor);
```

Comment:
Shouldn't that one here be executed nonetheless?

Reply:
Ah, good catch, thanks!